pyproject.toml

```diff
@@ -14,8 +14,8 @@ authors = [
 description = "Extremely lightweight compatibility layer between dataframe libraries"
 readme = "README.md"
 keywords = ["dataframes", "interoperability", "pandas", "polars", "pyarrow", "dask", "modin", "cudf"]
-license = {file = "LICENSE.md"}
 classifiers = [
+  "License :: OSI Approved :: MIT License",
```

Comment:
Should/Could both be kept at the same time?

Reply:
they recommend just using this https://packaging.python.org/en/latest/guides/writing-pyproject-toml/#license